hikaricp/datadog_checks/hikaricp/metrics.py

```diff
@@ -0,0 +1,16 @@
+METRIC_MAP = {
+    "hikaricp_connections": "connections",
+    "hikaricp_connections_active": "connections.active",
+    "hikaricp_connections_idle": "connections.idle",
+    "hikaricp_connections_pending": "connections.pending",
```

Comment:
We could make those metrics name more explicit:
```suggestion
    "hikaricp_connections": "connections.count.total",
    "hikaricp_connections_active": "connections.count.active",
    "hikaricp_connections_idle": "connections.count.idle",
    "hikaricp_connections_pending": "connections.count.pending",
```

Reply:
not sure this is relevant as cout type have a `count` suffix https://docs.datadoghq.com/integrations/openmetrics/#configuration

Reply:
I didn't thought about that.
My idea was the try to "group" metrics into some kind of categories:
- the number of connections (what I called `connections.count.*`)
- stats about usage: `connections.usage.*`
- stats about acquire: `connections.acquire.*`
- etc..

But it maybe it's better to keep the same naming than on openmetrics side.
So, closing this.

Reply:
Thanks for you comment @frivoire . Let's keep the openmetrics naming then.